api_tests/tests/reasonable_adjustment_flag/test_happy_cases.py

```diff
@@ -174,41 +214,49 @@ def test_flag_post(self):
             url=config.REASONABLE_ADJUSTMENTS_FLAG,
             headers={
                 'Authorization': f'Bearer {self.token}',
-                'nhsd-session-urid': 'test',
+                'nhsd-session-urid': '093895563513',
                 'x-request-id': str(uuid.uuid4()),
-                'content-type': 'application/fhir+json'
+                'content-type': 'application/fhir+json',
+                'Accept': 'application/fhir+json'
             },
-            json=json.dumps({'message': 'test'})
+            json=request_bank.get_body(Request.FLAG_POST),
         )
 
         # Then
         assert_that(expected_status_code).is_equal_to(response.status_code)
 
     @pytest.mark.happy_path
+    @pytest.mark.integration
     @pytest.mark.sandbox
     @pytest.mark.usefixtures('get_token_internal_dev')
     def test_flag_put(self):
+        # Pre-Req: Patient has both a consent and flag
+        Utils.send_consent_post(self.token)
+        Utils.send_flag_post(self.token)
+        get_flag_response = Utils.send_flag_get(self.token)
+
         # Given
         expected_status_code = 200
-        etag = Utils.get_etag(self,
-                              config.REASONABLE_ADJUSTMENTS_CONSENT,
-                              params={
-                                  'patient': '9999999998',
-                                  'category': 'test',
-                                  'status': 'test',
-                              })
+        flag_id = get_flag_response['id']
+        version_id = get_flag_response['version']
+
+        # todo on sandbox the consent_id and version_id cannot be None, need a cleaner way to do this
+        if self.sandbox is True:
```

Comment:
Gurdeep are sure it's not working with sandbox? Because sandbox GET /consent returns a real consent with `id` and and `versionId` so this test should work for both live and sandbox